data_structures/binary_search_tree.cpp

```diff
@@ -122,18 +117,35 @@ void Post(node *n) {
     }
 }
 
+int ReadInt() {
```

Comment:
What's the purpose of this new function? Would you mind explaining? Thanks. 🙂

Reply:
Hi! This function reads the variable and send an invalid option if the user writes a letter or other option not allowed in this program